sites/partners/package.json

```diff
@@ -47,14 +47,16 @@
     "clone-deep": "^4.0.1",
     "dayjs": "^1.10.7",
     "dotenv": "^17.2.3",
-    "markdown-to-jsx": "^7.7.12",
+    "mapbox-gl": "^3.16.0",
+    "markdown-to-jsx": "^7.7.16",
     "nanoid": "^3.1.12",
-    "next": "^14.2.28",
+    "next": "^15.5.7",
```

Comment:
praise: This new feature showing at the bottom left of all pages in dev mode is nice 

<img width="302" height="216" alt="Image" src="https://github.com/user-attachments/assets/dc30e180-dec3-4121-b6c5-c027f763a55e" />